proto/proto/geyser.proto

```diff
@@ -44,9 +28,6 @@ message PartialAccountUpdate {
 
   // AccountReplica version.
   uint32 replica_version = 7;
-
-  // timestamp the message was generated
-  google.protobuf.Timestamp ts = 8;
```

Comment:
this gonna break existing subscriber?

Reply:
looks like we never used this field in block engine